data_structures/src/chain.rs

```diff
@@ -2099,6 +2100,50 @@ impl UnspentOutputsPool {
     }
 }
 
+/// Information about our own UTXOs
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct UtxoInfo {
+    /// Vector of OutputPointer with their values
+    pub utxos: Vec<(OutputPointer, u64)>,
+    /// Counter of UTXOs bigger than the minimum collateral
+    pub bigger_than_min_counter: usize,
+    /// Counter of UTXOs bigger than the minimum collateral and older than collateral coinage
+    pub old_utxos_counter: usize,
```

Comment:
Maybe it would be better to calculate this in the client, not here. So for example users can see which UTXOs can be used for collateral, not just how many. We can do that by adding `included_in_block_number` to `utxos`, but we also need the `collateral_minimum` value.

Reply:
If this is too much we can open a new issue for adding features to `utxosInfo` :P

Reply:
I support that idea. The node itself should always give raw data over the JSONRPC interface. Then, we can process it and make it more meaningful to end users in the CLI. This also has the benefit of offloading the node server as any additional computation is done on the client side.

Reply:
Oks. Now I return utxo, value, time_lock and if it is ready to collaterize